tests/shakedown/shakedown/clients/metronome.py

```diff
@@ -15,17 +15,16 @@
 EMBED_HISTORY_SUMMARY = 'historySummary'
 
 
-def create_client(toml_config=None):
+def create_client(auth_token=None):
```

Comment:
I'm not sure I feel comfortable with removing toml_config... I get that it wasn't used... however client modules like this use the toml_config for service specific configurations... for instance this is used for marathon to determine which marathon to use root or mom... and specifically which mom.   This also enables specifying a locally running service (for instance marathon on a localhost).

Reply:
The idea of the refactoring is *not* to use the CLI toml config but see the CLI as a blackbox. The sessions for Metronome and Marathon should be managed without changing the toml. See https://github.com/mesosphere/marathon/blob/master/tests/system/common.py#L144 on master for an example. #6566 introduces the option to set the session with the CLI. This was done to ease daily debugging of system tests. The Jenkins jobs do not use the CLI sessions.